backend/cmd/headlamp.go

```diff
@@ -185,7 +187,7 @@ func serveWithNoCacheHeader(fs http.Handler) http.HandlerFunc {
 	}
 }
 
-// nolint:gocognit,funlen
+// nolint:gocognit,funlen,gocyclo
```

Comment:
We should make sure things are readable and these checks exist to help us figure that out. But we can do this as part of the backend improvements that @yolossn will work on.